vmdb/app/models/orchestration_stack_output.rb

```diff
@@ -3,4 +3,8 @@ class OrchestrationStackOutput < ActiveRecord::Base
 
   belongs_to :stack, :class_name => "OrchestrationStack"
   include ReportableMixin
+
+  def name
+    description
```

Comment:
I agree `key` is better. `key` is short and more like a name. `description` is optional.